tools/translations/fr.po

```diff
@@ -68,8 +68,8 @@ msgid ""
 "A texture with the shape of the light must be supplied to the 'texture' "
 "property."
 msgstr ""
-"Une texture avec la forme de la lumière doit être fournie dans la propriété « "
-"texture »."
+"Une texture avec la forme de la lumière doit être fournie dans la propriété "
```

Comment:
This diff is produced because your Poedit is configured (by default I think) to crop lines at 78 or 79 characters, while we use 80 characters as a default.

Do you think you could change this setting in your Poedit to 80 characters, resave the French po file with it and use ``git commit --amend`` to edit the current commit? You can then update the PR with ``git push --force origin master`` if all went well.

If not, it's no big deal, but having smaller diffs help proofread the changes :)